### DISCUSSION TITLE
On prem Jira v8.20.11 support

### DISCUSSION THREAD

**Discussion by norbix** [andygrunwald/go-jira#600] [Q&A]
Hi guys,

Do this SDK work with v8.20.11 ?

BR
/norbix

**Comment by andygrunwald** [comment]
@norbix Yes, it should.
It depends on the endpoints you wanna call and if they are properly implemented.
Can you describe the use case/endpoints you aim to call? Then the answer can be a bit more specific.

**Reply by norbix** [reply]
@andygrunwald Thx 4 the answer :) 

I'm able to interact with v8.20.11 with this SDK so that's perfect. 
No need for serialising into custom models anymore.

BTW,
Do u guys also support interaction with GraphQL endpoints e.g. seach ?

BR
/norbix

**Reply by andygrunwald** [reply]
@norbix Great to hear.
No, the GraphQL endpoint is currently not supported.
Can you open a feature request?
